src/sdsc/xslt.py

```diff
@@ -0,0 +1,396 @@
+#
```

Comment:
Yes, that's true. The problem is, I'm not sure if we can stuff all the extension functions into a class. This is something I have to think about more.

For the time being, I've used the more obvious solution.